tests/MyGame/MonsterExtra.cs

```diff
@@ -29,31 +29,16 @@ public struct MonsterExtra : IFlatbufferObject
   public bool MutateTestdPinf(double testd_pinf) { int o = __p.__offset(12); if (o != 0) { __p.bb.PutDouble(o + __p.bb_pos, testd_pinf); return true; } else { return false; } }
   public double TestdNinf { get { int o = __p.__offset(14); return o != 0 ? __p.bb.GetDouble(o + __p.bb_pos) : (double)Double.NegativeInfinity; } }
   public bool MutateTestdNinf(double testd_ninf) { int o = __p.__offset(14); if (o != 0) { __p.bb.PutDouble(o + __p.bb_pos, testd_ninf); return true; } else { return false; } }
+  public ArrayStruct? Testa { get { int o = __p.__offset(16); return o != 0 ? (ArrayStruct?)(new ArrayStruct()).__assign(o + __p.bb_pos, __p.bb) : null; } }
 
-  public static Offset<MonsterExtra> CreateMonsterExtra(FlatBufferBuilder builder,
```

Comment:
Is this constructor being deleted because we don't generate them for structs that have arrays in them? If so that is fine I guess.

Reply:
This constructor seems to be deleted as soon as a struct is added to a table. This is not related to the patch set.